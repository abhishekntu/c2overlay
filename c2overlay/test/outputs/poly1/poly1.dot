digraph graphname {
N8 [color="black", label="N6N3_add_N6_mul_N3_N8"];
N9 [color="black", label="O0_N12_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="sub_Imm_1_N11_N3"];
N4 [color="black", label="sqr_N8_N4"];
N5 [color="black", label="sqr_N9_N5"];
N6 [color="black", label="N4N7_add_Imm_1_N4_sub_N7_N6"];
N7 [color="black", label="N5N10_mul_N5_sub_N10_N7"];
N8 -> N7;
N1 -> N5;
N1 -> N6;
N1 -> N3;
N2 -> N8;
N2 -> N4;
N2 -> N6;
N3 -> N8;
N4 -> N7;
N5 -> N8;
N6 -> N7;
N7 -> N9;
}
