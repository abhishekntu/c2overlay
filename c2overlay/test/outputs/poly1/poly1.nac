procedure foo(in s32 x,in s32 y,out s32 D_2446)
{
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= add x,1;
  t1_2 <= sqr y;
  t2_1 <= sqr x;
  t2_2 <= sub x,1;
  t2_3 <= sub t1_1,y;
  t3_1 <= add t2_2,y;
  t3_2 <= mul t2_3,t1_2;
  t4_1 <= mul t2_1,t3_1;
  D_2446 <= sub t4_1,t3_2;
}
