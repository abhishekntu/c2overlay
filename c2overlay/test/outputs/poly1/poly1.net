.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.output out:N9
pinlist: N9
    
.clb N8_blk
pinlist: N3 N5 N2 open N8 open open open open 
subblock: N8_blk 0 1 2 open 4 open open open open
    
.clb N3_blk
pinlist: N1 open open open N3 open open open open 
subblock: N3_blk 0 open open open 4 open open open open

.clb N4_blk
pinlist: N2 open open open N4 open open open open 
subblock: N4_blk 0 open open open 4 open open open open

.clb N5_blk
pinlist: N1 open open open N5 open open open open 
subblock: N5_blk 0 open open open 4 open open open open

.clb N6_blk
pinlist: N1 N2 open open N6 open open open open 
subblock: N6_blk 0 1 open open 4 open open open open

.clb N7_blk
pinlist: N4 N6 N8 open N9 open open open open 
subblock: N7_blk 0 1 2 open 4 open open open open
    
