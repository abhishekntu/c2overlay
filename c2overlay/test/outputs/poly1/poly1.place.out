Netlist file: test.net   Architecture file: test.arch.xml
Array size: 3 x 3 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N1		1	0	0	#0
N2		3	0	0	#1
out:N9		4	1	0	#2
N8_blk		2	1	0	#3
N3_blk		1	1	0	#4
N4_blk		3	2	0	#5
N5_blk		1	2	0	#6
N6_blk		2	2	0	#7
N7_blk		3	1	0	#8
